src/System.Runtime.InteropServices.RuntimeInformation/tests/CheckPlatformTests.cs

```diff
@@ -5,20 +5,10 @@ namespace System.Runtime.InteropServices.RuntimeInformationTests
 {
     public class CheckPlatformTests
     {
-        [Fact, PlatformSpecific(PlatformID.Windows)]
-        public void CheckWindows()
-        {
-            Assert.True(RuntimeInformation.IsOSPlatform(OSPlatform.Windows));
-            Assert.True(RuntimeInformation.IsOSPlatform(OSPlatform.Create("WINDOWS")));
-            Assert.False(RuntimeInformation.IsOSPlatform(OSPlatform.Create("windows")));
-            Assert.False(RuntimeInformation.IsOSPlatform(OSPlatform.Create("Windows NT")));
-            Assert.False(RuntimeInformation.IsOSPlatform(OSPlatform.Linux));
-            Assert.False(RuntimeInformation.IsOSPlatform(OSPlatform.OSX));
-        }
-
         [Fact, PlatformSpecific(PlatformID.Linux)]
         public void CheckLinux()
         {
+            Assert.False(RuntimeInformation.IsOSPlatform(OSPlatform.FreeBSD));
```

Comment:
As I noted on the previous commits, I think we should leave the Asserts.True first followed by the Asserts.False.  I know you're trying to establish an alphabetical order here, but it's easier to read and understand (at least for me) when everything's grouped together by whether it should pass or not, and when the things that should pass all come first.